gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/InsertAllRequestTest.java

```diff
@@ -0,0 +1,161 @@
+          .build();
+  private static final InsertAllRequest INSERT_ALL_REQUEST7 =
+      InsertAllRequest.builder(TABLE_ID.dataset(), TABLE_ID.table(), ROWS_WITH_ID.get(0),
+          ROWS_WITH_ID.get(1))
+          .ignoreUnknownValues(IGNORE_UNKNOWN_VALUES)
+          .skipInvalidRows(SKIP_INVALID_ROWS)
+          .build();
+  private static final InsertAllRequest INSERT_ALL_REQUEST8 =
+      InsertAllRequest.builder(TABLE_ID.dataset(), TABLE_ID.table())
+          .addRow("id1", CONTENT1)
+          .addRow("id2", CONTENT2)
+          .ignoreUnknownValues(IGNORE_UNKNOWN_VALUES)
+          .skipInvalidRows(SKIP_INVALID_ROWS)
+          .build();
+
+  @Test
+  public void testBuilder() {
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST1.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST2.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST3.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST4.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST5.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST6.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST7.table());
+    assertEquals(TABLE_ID, INSERT_ALL_REQUEST8.table());
+    assertEquals(ROWS, INSERT_ALL_REQUEST1.rows());
+    assertEquals(ROWS, INSERT_ALL_REQUEST2.rows());
+    assertEquals(ROWS, INSERT_ALL_REQUEST4.rows());
+    assertEquals(ROWS, INSERT_ALL_REQUEST6.rows());
+    assertEquals(ROWS_WITH_ID, INSERT_ALL_REQUEST3.rows());
+    assertEquals(ROWS_WITH_ID, INSERT_ALL_REQUEST5.rows());
+    assertEquals(ROWS_WITH_ID, INSERT_ALL_REQUEST7.rows());
+    assertEquals(ROWS_WITH_ID, INSERT_ALL_REQUEST8.rows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST1.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST2.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST3.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST4.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST5.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST6.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST7.skipInvalidRows());
+    assertEquals(SKIP_INVALID_ROWS, INSERT_ALL_REQUEST8.skipInvalidRows());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST1.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST2.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST3.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST4.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST5.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST6.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST7.ignoreUnknownValues());
+    assertEquals(IGNORE_UNKNOWN_VALUES, INSERT_ALL_REQUEST8.ignoreUnknownValues());
+  }
+
+  @Test
+  public void testOf() {
```

Comment:
nit: I think two of the `of` methods with varargs in the method signature aren't tested